frontend/src/components/layouts/WideLayout.js

```diff
@@ -97,6 +98,7 @@ export default function WideLayout({
         transparentHeader={transparentHeader}
         noSpacingBottom={isStaticPage}
         background={headerBackground}
+        isHubPage={isHubPage}
```

Comment:
Hey @JoshKornfeld this is a common WideLayout page that is used by multiple other pages. I don't think it is good a idea to add Hub specific arguments here. cc @positiveimpact

Reply:
I think since we also have `isStaticPage` and hubs are quite central to our concept, this is not a huge issue imo